adds a PoC issue template (#665)
kyma-project/infrastructure-manager: .github/ISSUE_TEMPLATE/poc.md

---
name: Proof of Concept 
about: Suggest an improvement to the project
labels: ["area/control-plane", "kind/feature"]
projects: ["kyma-project/24"]
---

<!-- Thank you for your contribution. Before you submit the issue:
1. Search open and closed issues for duplicates.
2. Read the contributing guidelines.
-->

**Description**
<!-- Provide a clear and concise description of the feature. -->
This is a Proof of Concept. Here's a checklist that should help you consider often neglected aspects:-
- [ ] should the PoC include any documentation such as ADR?
- [ ] is it clear who will benefit from the PoC and how?
- [ ] will the final implementation require any migration steps?
- [ ] is there a need for alignment with KEB/SRE/other teams? 

> - [ ] usually it's worth to timebox such PoC tasks. What would a good timebox in this case?

Reviewer: ```suggestion
- [ ] usually it's worth timeboxing such PoC tasks. What would be a good timebox in this case?
```

- [ ] will the final solution need a new repository or any pipelines adjustments?

> - [ ] are we already aware about any additional challenges related to release process? E.g. should an additional threat modelling sessions be scheduled?

Reviewer: ```suggestion
- [ ] are we already aware of any additional challenges related to the release process? E.g. should additional threat modelling sessions be scheduled?
```
